src/metalnx-web/src/main/java/com/emc/metalnx/interceptors/HttpResponseHandlerInterceptor.java

```diff
@@ -66,14 +67,14 @@ public void postHandle(final HttpServletRequest request, final HttpServletRespon
 			modelAndView.getModelMap().addAttribute("emcmetalnxVersion", emcmetalnxVersion);
 			modelAndView.getModelMap().addAttribute("jargonVersion", new JargonVersion());
 			modelAndView.getModelMap().addAttribute("globalConfig", configService.getGlobalConfig());
-
-			DataGridUser loggedUser = loggedUserUtils.getLoggedDataGridUser();
-
-			logger.debug("dataGridUser:{}", loggedUser);
-
-			modelAndView.getModelMap().addAttribute("dataGridUser", loggedUser);
-
+			
+			// We are no longer using the admin login credentials. If this runs before login the auth will
```

Comment:
The first statement could lead to some readers of the code to think that admin credentials are used in other places, just not here. I think we need to reword the comment so that there's no room for assumptions.

Reply:
I reworded it.  Please take a look.

Reply:
The updated wording is much better.